actors/datacap/src/lib.rs

```diff
@@ -71,14 +72,14 @@ pub struct Actor;
 
 impl Actor {
     /// Constructor for DataCap Actor
-    pub fn constructor(rt: &mut impl Runtime, governor: Address) -> Result<(), ActorError> {
+    pub fn constructor(rt: &mut impl Runtime, params: Address) -> Result<(), ActorError> {
```

Comment:
This rename reduces clarity and doesn't seem necessary

Reply:
Sure. I was hoping to have all methods uniformly take a single parameter named `params` (or no parameters at all), but I'm not fussed.

Reply:
I think `params` is fine if we've wrapped in a transparent struct, so the struct member then has an informative name.